examples/CH4-refact-dpdispatcher/CH4_init_bulk.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
It's unclear to me why you need to put a param file under the new dpdispatcher directory? I think the machine file is enough?

Reply:
You are right. I create this dir for quickly hands on.
Maybe I could use soft link instead of the json file.